src/trade/tradeConf.es6

```diff
@@ -155,7 +155,7 @@ const registerTicks = (state, extra) => {
 
       if (should_add_new_tick) {
          const contract_is_finished = proposal_open_contract.status !== 'open' && (tick_count < -1 || is_path_dependent_last_tick);
-         state.buy.barrier = proposal_open_contract.barrier ? (+proposal_open_contract.barrier) : null;
+         state.buy.barrier = proposal_open_contract.barrier ? addComma(+proposal_open_contract.barrier, display_decimals) : null;
```

Comment:
This will cause the barrier to not be drawn on the chart. One option would be to add display_decimals to the state and pass it to the `add-comma` rivets formatter